src/encoded/tests/test_audit_functional_characterization_experiment.py

```diff
@@ -202,9 +202,8 @@ def test_audit_experiment_replicated(
         {'status': 'submitted', 'date_submitted': '2015-03-03'}
     )
     res = testapp.get(pooled_clone_sequencing['@id'] + '@@index-data')
-    assert any(
-        error['category'] == 'unreplicated experiment' and
-        error['level_name'] == 'WARNING'
```

Comment:
The reason it used to be WARNING is from these two lines: https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/audit/experiment.py#L1535-#L1536

I know auditing of a PCS should no longer reach that section because it will exit at control_type, but those lines in audit/experiment.py should be removed for completeness

Reply:
removed the lines in experiment audit